vcbuild.bat

```diff
@@ -81,7 +84,6 @@ goto next-arg
 
 :args-done
 if defined upload goto upload
-if defined jslint goto jslint
```

Comment:
Does `vcbuild jslint` still work? It seems that with this change it would only work when tests are run too.